src/pages/workspaces/workspace/notebooks/NotebookLauncher.js

```diff
@@ -225,8 +227,19 @@ class NotebookEditor extends Component {
       }))
 
       const { name: workspaceName, app } = this.props
-      if (app === 'lab') this.setState({ url: `${clusterUrl}/${app}/tree/${workspaceName}/${notebookName}` })
-      else this.setState({ url: `${clusterUrl}/notebooks/${workspaceName}/${notebookName}` })
+      if (app === 'lab') {
+        this.setState({ url: `${clusterUrl}/${app}/tree/${workspaceName}/${notebookName}` })
+        pushNotification({
+          type: 'warning',
+          dismissable: { click: true },
+          dismiss: { duration: 30000 },
+          content: h(pushNotificationMessage, {
+            title: 'Autosave every 2 minutes',
+            content: 'JupyterLab autosaves every 2 min. Please be extra careful about saving your notebook before exiting the window.'
```

Comment:
Is there a way that users can force-save their notebook before exiting? Can they save their notebook in the menu or with a keyboard shortcut? If so then I'd recommend explicitly telling them to Save your notebook before closing this notebook or window, or something like that.

Reply:
Yeah, you can command+S or press the save button to force save. I'll change the messaging to just tell them to save before exiting